tests/testJson.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+#
+# LSST Data Management System
+# Copyright 2012-2016 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import print_function
+
+import os
+import sys
+import tempfile
+import unittest
+
+import numpy as np
+
+import lsst.utils.tests as utilsTests
+
+import lsst.pipe.base as pipeBase
+from lsst.validate.drp.io import saveKpmToJson
+
+
+class JsonTestCase(unittest.TestCase):
+    """Testing basic coordinate calculations."""
+
+    def testSaveJson(self):
+        ps = pipeBase.Struct(foo=2, bar=[10, 20], hard=np.array([5,10]))
+        _, tmpFilepath = tempfile.mkstemp(suffix='.json')
+        saveKpmToJson(ps, tmpFilepath)
+        os.unlink(tmpFilepath)
```

Comment:
I'll just note that if the routine explicitly says "saving to JSON" then it should be readable by any JSON reader. If the routine was "saveKpm" then the format would implicitly be a black box and a "loadKpm" routine would be needed.